packages/order-utils/src/formatters.ts

```diff
@@ -0,0 +1,23 @@
+import { Order, OrderAddresses, OrderValues } from '@0xproject/types';
+import { BigNumber } from '@0xproject/utils';
+
+export const formatters = {
+    getOrderAddressesAndValues(order: Order): [OrderAddresses, OrderValues] {
+        const orderAddresses: OrderAddresses = [
+            order.maker,
+            order.taker,
+            order.makerTokenAddress,
+            order.takerTokenAddress,
+            order.feeRecipient,
+        ];
+        const orderValues: OrderValues = [
+            order.makerTokenAmount,
+            order.takerTokenAmount,
+            order.makerFee,
+            order.takerFee,
+            order.expirationUnixTimestampSec,
+            order.salt,
+        ];
+        return [orderAddresses, orderValues];
```

Comment:
**var-return**

Reply:
I think this is more readable then assigning it to some non-descript variable name. Don't know what I would call this.